script/Arcueid/Day 11/11_00_ARC11_3.txt

```diff
@@ -1450,7 +1450,7 @@ Ah. I'm sorry, I'm all too aware of that myself.
 -- Page 247, Offset 30222.
 -- 「……でも、可笑しいね。殺した男と殺された女が、こうして抱き合ってるなんて」
 // boy and girl for a maybe failed "boy meets girl" allusion? this is kinda cheesy in a funny way i like it
-"But... It's funny isn't it? Killer boy and victim girl embracing each other like this?"
+"But... it's funny, isn't it? Killer boy and victim girl embracing each other like this?"
```

Comment:
I think `isn't it?` should be without a question mark.

Reply:
I dunno, I kinda like this one. Makes her sound a little wistful.